ste/sender-blockBlobFromURL.go

```diff
@@ -163,6 +178,32 @@ func (c *urlToBlockBlobCopier) generatePutBlockFromURL(id common.ChunkID, blockI
 	})
 }
 
+func (c *urlToBlockBlobCopier) generateStartCopyBlobFromURL(id common.ChunkID, blockIndex int32, adjustedChunkSize int64) chunkFunc {
+	return createSendToRemoteChunkFunc(c.jptm, id, func() {
+
+		encodedBlockID := c.generateEncodedBlockID()
+
+		c.setBlockID(blockIndex, encodedBlockID)
```

Comment:
block ID is not needed if we are just calling CopyFromURL, right?

Reply:
Removed the blockID generation